client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -84,6 +84,27 @@
             "defaultExistingProject": true
           }
         ]
+      },
+      "secondaryKeyAsGSI": {
+        "description": "Changes the behaviour of @key to remove the creation of an LSI.",
```

Comment:
I'd suggest: "Changes the behavior of @key to always create GSIs"

Reply:
Changes the @key behavior to always create GSIs.